### PR TITLE
Add support for GEOIDMODEL

### PR DESCRIPTION
**Pull request by rouault**
- WKT and PROJJSON: add import/export of geoid model of VertCRS
-  Add a geoid_model name in database, use GEOIDMODEL for transformations, add a proj_create_vertical_crs_ex()
-  Database: add mapping between EPSG geoid and PROJ grid names for GEOID99, 03, 06, 09 and 18. Related to OSGeo/proj-datumgrid#60 that add those grids in proj-datumgrid-north-america

### CODE REVIEW

File: schemas/v0.2/projjson.schema.json [hunk 1]
```diff
@@ -0,0 +1,981 @@
+{
+  "$id": "https://proj.org/schemas/v0.2/projjson.schema.json",
```
Comment (kbevers) [inline schemas/v0.2/projjson.schema.json:2]: `docs/source/usage/projjson.rst` needs to be updated to version 0.2. I it not immediately clear to me if there's version 0.2 features that needs to be highlighted in the docs.

File: src/iso19111/c_api.cpp [hunk 2]
```diff
@@ -2934,15 +2940,73 @@ PJ *proj_create_vertical_crs(PJ_CONTEXT *ctx, const char *crs_name,
                              const char *datum_name, const char *linear_units,
                              double linear_units_conv) {
 
+    return proj_create_vertical_crs_ex(
```
Comment (kbevers) [inline src/iso19111/c_api.cpp]: What does the `_ex` part of the name mean?

File: src/iso19111/c_api.cpp [hunk 3]
```diff
@@ -2934,15 +2940,73 @@ PJ *proj_create_vertical_crs(PJ_CONTEXT *ctx, const char *crs_name,
...
+ * @param geoid_model_auth_name Authority name of the transformation for
+ * the geoid model. or NULL
+ * @param geoid_model_code Code of the transformation for
+ * the geoid model. or NULL
+ * @param geoid_geog_crs Geographic CRS for the geoid transformation, or NULL.
+ * @param options should be set to NULL for now
+ * @return Object of type VerticalCRS that must be unreferenced with
+ * proj_destroy(), or NULL in case of error.
+ */
+PJ *proj_create_vertical_crs_ex(
```
Comment (kbevers) [inline src/iso19111/c_api.cpp:2990]: What does the `_ex` part of the name mean?

**Review by kbevers — APPROVED** [review]
This is a lot to grasp but i believe it looks okay. I have added a few inlined comments.

File: src/iso19111/c_api.cpp (hunk 3 above)
Comment (rouault) [reply src/iso19111/c_api.cpp:2990]: For extended

File: schemas/v0.2/projjson.schema.json (hunk 1 above)
Comment (rouault) [reply schemas/v0.2/projjson.schema.json:2]: added in a new commit, with a history section

File: src/iso19111/c_api.cpp (hunk 3 above)
Comment (kbevers) [reply src/iso19111/c_api.cpp:2990]: Okay. Would it make sense to add a short text on what that entails to the documentation?

Comment (rouault) [reply src/iso19111/c_api.cpp:2990]: sure, added

**Pull request merged**